Assets/Script/ZoomCamera.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ZoomCamera : MonoBehaviour
+{
+
+    public HandlerCamera mainCamera;
```

Comment:
this is why I dont like the `handler` word cuz you are changing the distance of a camera not the `distance` of a handler